.gitignore

```diff
@@ -129,3 +129,7 @@ sklearn/svm/bounds.py
 sklearn/svm/libsvm.py
 sklearn/svm/libsvm_sparse.py
 sklearn/svm/liblinear.py
+
+sklearn/feature_extraction/dict_vectorizer.py
+sklearn/feature_extraction/hashing.py
```

Comment:
You will also need `_hashing_fast.py` here

Reply:
In this case:

1. `_hashing.pyx` -> `_hashing_fast.pyx`
2. `hashing.py` -> `_hashing.py`

And the `hashing.py` will import from `_hashing.py`.